src/Apps/Order/Components/TransactionDetailsSummaryItem.tsx

```diff
@@ -100,11 +100,11 @@ export const TransactionDetailsSummaryItem: FC<TransactionDetailsSummaryItemProp
       if (selectedShippingQuote) {
         return `${
           shippingQuoteDisplayNames[selectedShippingQuote.typeName]
-        } delivery`
+        } delivery*`
```

Comment:
I'm wondering if this `*` is going to show for all orders and not just make offer ones?


As Im going through the flow I see the `*` here for buy now orders which Im not sure we want since Shipping is not really negotiable here and there is no footnote at the bottom
<img width="898" alt="Screen Shot 2023-05-18 at 4 38 40 PM" src="https://github.com/artsy/force/assets/12748344/a7514425-9174-43c6-89c4-a9f21a3df23b">